jwst/resample/resample_spec_step.py

```diff
@@ -18,6 +18,11 @@ class ResampleSpecStep(ResampleStep):
         A singe datamodel, a container of datamodels, or an association file
     """
 
+    # Spec is all the same except for the suffix
+    spec = """
+        suffix = string(default='s2d')
```

Comment:
So are these now being set in both the calling pipeline module (e.g. `calwebb_spec2`, `calwebb_spec3`) and here in the step's `spec`?  I guess that covers cases where someone only runs the step and not the whole pipeline?

But if that's the case, why aren't we including `suffix` in the `spec` of lots more steps?

And does that mean that if someone runs the step by itself, the output file suffix will be `s2d`, instead of the usual `resample_spec`?

Reply:
Nope, good catch. The suffix should not be set here but by the pipelines that use this step. Will need to fix that.